Lua/Lua.sublime-syntax

```diff
@@ -102,7 +102,7 @@ contexts:
       push: expression-list
 
     - match: local{{identifier_break}}
-      scope: storage.modifier.lua
+      scope: storage.modifier.lua keyword.declaration.local.lua
```

Comment:
Should `local` really get the `keyword.declaration` scope? According to the guidelines, this applies only to function/class/type definition keywords (`storage.type`), but not to modifiers (`storage.modifier`).

Reply:
I think you are correct, the motivation of doing this was to fix syntax highlighting inconsistencies with the **local** keyword on some color themes, but I've now realized that actually the color scheme was doing wrong. The change was removed.